components/Link/index.js

```diff
@@ -155,21 +150,17 @@ const Link = ({
 	return (
 		<>
 			<LinkOutput
-				// eslint-disable-next-line react/jsx-props-no-spreading
-				{...blockProps}
 				tagName="a"
 				identifier="label"
-				className="tenup-block-components-link__label"
+				className={classnames('tenup-block-components-link__label', className)}
 				value={value}
 				onChange={onTextChange}
 				aria-label={__('Link text', '10up-block-components')}
 				placeholder={placeholder}
-				withoutInteractiveFormatting
```

Comment:
Since we are already setting the `allowedFormats` this had no effect. 

The `withoutInteractiveFormatting` removes any rich text formats that add interactivity to the string.